src/model/PackageVersion.ts

```diff
@@ -18,23 +18,24 @@ export class PackageVersion {
         const packagePath = path.resolve(mainRepo, PackageVersion.PACKAGE_JSON);
         if (!fs.existsSync(packagePath)) {
             console.warn(cwarn`[NPM] Could not find package.json in repo ${mainRepo}...`);
-            return null;
```

Comment:
So now the `_version` cannot be `null` anymore. (See Line 8).

Reply:
Thanks, updated it! :)